src/main/scala/com/redislabs/provider/redis/redisFunctions.scala

```diff
@@ -34,6 +34,22 @@ class RedisContext(@transient val sc: SparkContext) extends Serializable {
     new RedisKeysRDD(sc, redisConfig, readWriteConfig, keyPattern, partitionNum, null)
   }
 
+  /**
+    * @param keysOrKeyPattern an array of keys or a key pattern
+    * @param partitionNum number of partitions
+    * @return RedisHashRDD of related Key-Values stored in redis server
+    */
+  def fromRedisHLL[T](keysOrKeyPattern: T,
+                      partitionNum: Int = 3)
+                     (implicit redisConfig: RedisConfig = new RedisConfig(new RedisEndpoint(sc.getConf))):
+  RDD[(String, Long)] = {
+    keysOrKeyPattern match {
+      case keyPattern: String => fromRedisKeyPattern(keyPattern, partitionNum)(redisConfig).getPFCount()
```

Comment:
redisConfig is an implicit, so no need to specify it explicitly (just to follow other methods like this, e.g. `fromRedisKV`)